prow/plugins/dco/dco.go

```diff
@@ -162,11 +228,39 @@ func checkExistingLabels(gc gitHubClient, l *logrus.Entry, org, repo string, num
 
 // takeAction will take appropriate action on the pull request according to its
 // current state.
-func takeAction(gc gitHubClient, cp commentPruner, l *logrus.Entry, org, repo string, pr github.PullRequest, commitsMissingDCO []github.GitCommit, existingStatus string, hasYesLabel, hasNoLabel, addComment bool) error {
+func takeAction(gc gitHubClient, cp commentPruner, l *logrus.Entry, org, repo string, pr github.PullRequest, commitsMissingDCO []github.GitCommit, existingStatus string, hasYesLabel, hasNoLabel, addComment, trustedOrgMember bool) error {
```

Comment:
Could we not pass around pointers to the config please, and make a lack of configuration equal to the default case?

Reply:
> Could we not pass around pointers to the config please
Fixed, we're not passing pointers anymore
> make a lack of configuration equal to the default case
What is default case supposed to be here? Do we want default case to be that if config is not provided, always do DCO checks?

Reply:
The default case should be the same behavior as before this change -- to get the new behavior you must opt-in

Reply:
Makes sense. To solve this I added the `SkipDcoCheck` field that is by default false but if operator wants to use this feature `SkipDcoChecks` needs to be set to true. Please let me know if you have any better idea.